packages/react-composites/src/composites/CallWithChatComposite/components/DialpadModal.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useState } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useMemo } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Dialpad, DialpadStyles, _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { IButtonStyles, IModalStyles, Modal, Stack, useTheme, Text, IconButton, PrimaryButton } from '@fluentui/react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Call20Regular } from '@fluentui/react-icons';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { drawerContainerStyles } from '../styles/CallWithChatCompositeStyles';
+
+/** @private */
+export interface DialpadModalStrings {
+  dialpadModalAriaLabel: string;
+  DialpadModalTitle: string;
+  closeModalButtonAriaLabel: string;
+  callButtonLabel: string;
+}
+
+/** @private */
+export interface DialpadModalProps {
+  isMobile: boolean;
+  isModalOpen: boolean;
+  strings: DialpadModalStrings;
+  hideModal: () => void;
```

Comment:
Rename to match our convention of naming handlers with `on` prefix: `onDismiss` (`Modal` is redudant).